.github/workflows/lambdachecks.yml

```diff
@@ -0,0 +1,25 @@
+name: Lambda checks
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7]
```

Comment:
Should we also add 3.7 now that the lambda is py3 compatible ?

Reply:
Added but restricted the checks to logs-monitoring for now